paket.lock

```diff
@@ -21,3 +21,35 @@ NUGET
     xunit.extensibility.execution (2.3.1)
       xunit.extensibility.core (2.3.1) - restriction: || (&& (== net20) (>= netstandard1.1)) (&& (== net40) (>= netstandard1.1)) (== net452)
     xunit.runner.visualstudio (2.3.1) - version_in_path: false
+
+GROUP build
+RESTRICTION: || (== net20) (== net40) (== net452)
+NUGET
+  remote: https://www.nuget.org/api/v2
+    Chessie (0.6) - restriction: || (&& (== net20) (>= net45)) (&& (== net20) (>= netstandard2.0)) (&& (== net40) (>= net45)) (&& (== net40) (>= netstandard2.0)) (== net452)
+      FSharp.Core
+    FAKE (4.64.11)
+    FSharp.Compiler.Tools (10.0.1) - restriction: || (&& (== net20) (>= net45)) (&& (== net40) (>= net45)) (== net452)
+    FSharp.Core (4.3.4) - restriction: || (&& (== net20) (>= net45)) (&& (== net40) (>= net45)) (== net452)
+    Mono.Cecil (0.10.0-beta7) - restriction: || (&& (== net20) (>= net45)) (&& (== net20) (>= netstandard2.0)) (&& (== net40) (>= net45)) (&& (== net40) (>= netstandard2.0)) (== net452)
+    Newtonsoft.Json (11.0.1) - restriction: || (&& (== net20) (>= net45)) (&& (== net40) (>= net45)) (== net452)
+    NUnit.Console (3.8)
+      NUnit.ConsoleRunner (>= 3.8)
+      NUnit.Extension.NUnitProjectLoader (>= 3.5)
+      NUnit.Extension.NUnitV2Driver (>= 3.6)
+      NUnit.Extension.NUnitV2ResultWriter (>= 3.5)
+      NUnit.Extension.TeamCityEventListener (>= 1.0.3)
+      NUnit.Extension.VSProjectLoader (>= 3.7)
+    NUnit.ConsoleRunner (3.8)
+    NUnit.Extension.NUnitProjectLoader (3.6)
+    NUnit.Extension.NUnitV2Driver (3.7)
+    NUnit.Extension.NUnitV2ResultWriter (3.6)
+    NUnit.Extension.TeamCityEventListener (1.0.3)
+    NUnit.Extension.VSProjectLoader (3.7)
+    Paket.Core (5.151.4)
```

Comment:
this all looks like a lot of stuff, and paket bootstrap + restore is responsible for getting fake so I think this should not live in here (or in the .dependencies, ideally?) I maye def be wrong; if this is for a good reason, please disregard

Reply:
Sure but now the runner + unit test framework are getting from the build script not by the appveyor runtime. For nunit I follow the reco to use the nunit runner console but it came with this dependencies.

In general it is better to have nuget tools rhat come from rhe build so we can share the script for appveyor + travis

Reply:
OK, this is all rooted in my misunderstanding of the paket bootstrapping. As long as the batch files are managing the load of the fake+runners+dependences (i.e. paket.core for packing) before the running of the fake (which I now see it does consistently), this all looks good.